libralgo/libral.go

```diff
@@ -1,4 +1,4 @@
-package libral
+package libralgo
```

Comment:
I don't understand your comment here, if you're talking about squashing commits then I think the recent push should address this.